src/encoded/static/components/status.js

```diff
@@ -110,6 +110,8 @@ const defaultObjectStatuses = {
             'archived',
             'revoked',
         ],
+        unprivileged: [
+        ],
```

Comment:
Is this still needed? Sort of odd this is the only use of `unprivileged` in this file, but it’s empty.